gstreamer/src/toc.rs

```diff
@@ -214,7 +221,7 @@ impl TocEntryRef {
                 return None;
             }
 
-            Some(TagList::from_glib_borrow(tags as *const ffi::GstTagList))
+            Some(TagList::from_glib_none(tags as *const ffi::GstTagList))
```

Comment:
Yes, that's why I said it seems unneeded :) It will work, but just keep it as-is for now.